extras/splat/splat-instance.js

```diff
@@ -32,11 +32,12 @@ class SplatInstance {
 
     lastCameraDirection = new Vec3();
 
-    constructor(splat, cameraEntity, debugRender = false) {
+    constructor(splat, options) {
```

Comment:
Might be worth documenting these options.

Reply:
Yep. At the moment none of the splat code has jsdocs, so that's all for a separate PR.